src/api/shop.js

```diff
@@ -65,21 +66,23 @@ export default {
   },
   getStores (address, cb) {
     Loading.show()
-    axios.get(SEARCH, {
-      params: {
-        lon: address.longitude,
-        lat: address.latitude,
-        time: 900
-      }
-    }).then(response => {
-      Loading.hide()
-      cb(response.data)
-    }).catch(function (error) {
-      console.log(error)
-      const alert = Alert.create({html: error.response.data.message, color: 'red-7'})
-      setTimeout(alert.dismiss, 5000)
-      Loading.hide()
-    })
+    if (address.longitude && address.latitude) {
```

Comment:
lets not reach out till we have a location or we get unnecessary 500's and additional load on the servers